packages/common-ui/src/InfoText.tsx

```diff
@@ -11,16 +11,22 @@
 //    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
-//@ts-nocheck
-import React from 'react';
+import { ReactNode } from 'react';
 
 import { makeStyles } from 'tss-react/mui';
 
 const useStyles = makeStyles()(theme => ({
   default: { color: theme.palette.text.disabled, margin: '15px 0' }
 }));
 
-export const InfoText = ({ children, className = '', variant = '', ...props }) => {
+interface InfoTextProps {
+  className?: string;
+  children: ReactNode;
+  variant?: 'dense' | '';
```

Comment:
I think in this case is better keep it an optional prop and remove the default param ` = ''`